ui/admin/app/components/mapping-list/index.hbs

```diff
@@ -36,20 +38,21 @@
                 {{on 'input' (set-from-event option 'value')}}
                 as |F|
               >
-                <F.Label>{{t 'form.value.label'}}</F.Label>
+                <F.Label class='sr-only'>{{t 'form.value.label'}}</F.Label>
               </Hds::Form::TextInput::Field>
             </B.Td>
-            <B.Td>
-              <Hds::Button
-                @text={{t 'actions.remove'}}
-                @color='critical'
-                @icon='trash'
-                @isIconOnly={{true}}
-                disabled={{@disabled}}
-                {{on 'click' (fn this.removeOptionByIndex index)}}
-              />
-
-            </B.Td>
+            {{#if @removeOptionByIndex}}
```

Comment:
Should the display of these cells be controlled by whether we have the function? The reason why I had the indirect function call was to guard against not including the function but I think it should still be required.

Wouldn't this also not work with the `last-child` css that shrinks the last cell?

Reply:
Good point.

Reply:
Good to know, wasn't aware of assert usage